docs/external-disk.md

```diff
@@ -19,45 +19,108 @@ The `-disk` specification may be repeated for multiple disks, although a limited
-2. `mount` which mounts the filesystem to a provided path
+```
+onboot:
+  name: format
+  image: linuxkit/format:${hash}
+```
 
-```yml
+In this mode of operation, the first disk found that does not have a valid partition table 
+will have one linux partition created that fills the entire disk
+
+### Options
+
+```
 onboot:
-  - name: format
-    image: linuxkit/format:84a997e69051a1bf05b7c1926ab785bb07932954
-  - name: mount
-    image: linuxkit/mount:ac8939c4102f97c084d9ddfd445c1908fce6d768
-    command: ["/mount.sh", "/var/external"]
+  name: format
+  image: linuxkit/format:${hash}
+  command: ["/usr/bin/format", "-type", "ext4", "-label", "DATA", "/dev/vda"]
 ```
 
-Notice several key points:
+`-type` can be used to specify the type e.g ext4
+`-label` can be used to give the disk a label
+The final (optional) argument specifies the device name
 
-1. format container
-    * The format container needs to have bind mounts for `/dev`
-    * The format container needs `CAP_SYS_ADMIN` and `CAP_MKNOD` capabilities
-    * The format container only needs to run **once**, not matter how many external disks or partitions are provided. It finds all block devices under `/dev` and processes them.
-    * The default container config should be sufficient
-2. mount container
-    * The mount container `command` is `mount.sh` followed by the desired mount point. Remember that nearly everything in a linuxkit image is read-only except under `/var`, so mount it there.
-    * The mount container needs to have bind mounts for `/dev` and `/var`
-    * The mount container needs `CAP_SYS_ADMIN` capabilities
-    * The mount container needs `rootfsPropagation: shared`
-    * The default container config should be sufficient, though the `mount.sh` command needs to be specified
+## Mount the disk
 
-With the above in place, if run with the current disk options, the image will make the external disk available as `/dev/vda1` and mount it at `/var/external`.
+Once a disk has been prepared it will need to be mounted using `pkg/mount`
+
+### Usage
+
+**NOTE: Block devices may only be mounted in `/var` unless you have explicitly added an additional bind mount**
+
+In this mode of operation, the first unmounted linux partition on the any block device is mounted to the mountpoint provided.
+
+```
+onboot:
+  name: mount
+  image: linuxkit/mount:${hash}
+  command: ["/usr/bin/mountie", "/var/lib/docker"]
+```
+
+### Options
+
+You can provide either a partition label, device name or disk UUID to specify which disk should be used.
+For example:
+
+```
+onboot:
+  name: mount
+  image: linuxkit/mount:${hash}
+  command: ["/usr/bin/mountie", "-label", "DATA", "/var/lib/docker" ]
+```
+
+```
+onboot:
+  name: mount
+  image: linuxkit/mount:${hash}
+  command: ["/usr/bin/mountie", "-uuid", "a-proper-uuid", "/var/lib/docker" ]
+```
+
+```
+onboot:
+  name: mount
+  image: linuxkit/mount:${hash}
+  command: ["/usr/bin/mountie", "-device", "/dev/sda1", "/var/lib/docker" ]
+```
```

Comment:
I know it seems painfully obvious, but if someone wants to format or mount multiple disks, they should just run the `format` container or `mount` container multiple times, correct? We probably should say so and give examples. You _know_ that someone will ask on the channel (or even submit a PR) with all sorts of syntaxes for formatting or mounting multiple disks at once. We even tried to do it here. Head that one off.